main.py

```diff
@@ -26,59 +31,79 @@ def validate_number(num):
         return False
 
 
-def validate_operator(operator):
+def validate_operator(operator: any) -> bool:
+    """Validate value for being in a list of operators and print a massage if the validation failed."""
     if operator in valid_operators:
         return True
 
     print(f'Please enter one on these operators: {", ".join(valid_operators)}')
     return False
 
 
-def handle_input(phrase, validator):
+def handle_input(phrase: str, validator: Callable[[any], bool]) -> T:
+    """
+    Wait for user's input, exit app or validate and return input
+
+    :param phrase: guiding message for the user
+    :param validator: function to validate user input
+    :return: valid user input
+    """
     res = input(phrase)
 
     if res == 'exit':
         app_exit()
-        return
 
     if not validator(res):
         res = handle_input(phrase, validator)
 
     return res
 
 
-def inputs():
+def read_user_inputs() -> None:
+    """Ask user to enter 2 numbers and operators, process them and show result, rinse and repeat"""
     a = handle_input("Enter first number:", validate_number)
     operator = handle_input("Enter the operator you want to use:", validate_operator)
     b = handle_input("Enter second number:", validate_number)
     count(a, b, operator)
+    read_user_inputs()
+
 
+def count(a: T, b: T, operator: str) -> None:
+    """
+    Perform math operation based on user inputs and print result
 
-def count(a, b, operator):
+    :param a: first number
+    :param b: second number
+    :param operator: math operator
+    """
     first_num = float(a)
     second_num = float(b)
```

Comment:
This is sth that confirms my previous comment. Despite you've declared that count() accepts argument 'a' that can be [int, float], you need to cast it again to float.

In this moment I'd like to suggest a bit other approach for calculating the result. Check this out:
![image](https://user-images.githubusercontent.com/33527461/129558156-35a89de8-22ba-4694-957e-4b1e33ef8f97.png)

the eval() function does the job for you. also checking for division by 0 is being done automatically by catching the exception. The only thing that left here is to change the way you present the outcome.
What do you think?